From c75b51a5f34ff016d0d96de90b1da21c4e338a1f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 8 Mar 2018 22:58:21 -0800
Subject: [PATCH] TRITON-220 TRITON-184 causes deadlock between registrar and
 config-agent in sapi which blocks startup of all (zone & GZ) config-agent's
 and many services

---
 bin/agent.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/agent.sh b/bin/agent.sh
index 194d8fc..1875885 100755
--- a/bin/agent.sh
+++ b/bin/agent.sh
@@ -38,8 +38,8 @@ CONFIG_FILE=
 POLL_INTERVAL_S=120
 
 SMF_INST=${SMF_FMRI##svc:*:}
-RUN_DIR=/var/run/config-agent-$SMF_INST
-FIRST_RUN_FILE=$RUN_DIR/first-run
+RUN_DIR=/var/config-agent
+FIRST_RUN_FILE=$RUN_DIR/$SMF_INST.first-run
 
 
 # ---- support functions
-- 
2.21.0

